### PR TITLE
Adds Ctrl+W shortcut, removes tweeting, removes build warnings

### PR DESCRIPTION
**Pull request by mahfiaz**
Adds Ctrl+W shortcut to close current tab in addition to Ctrl+F4.
Removes tweeting with suggested texts in PDF viewer.
Fixes whitespace in PDF viewer and remove some build warnings.

### CODE REVIEW

File: Utilities/GUI/DualTabbedLayoutStuff/DualTabbedLayout.cs [hunk 1]
```diff
@@ -696,7 +696,7 @@ private void window_KeyDown(object sender, KeyEventArgs e)
         {
             Window window = (Window)sender;
             DualTabbedLayoutItem item = (DualTabbedLayoutItem)window.Tag;
-            if (e.Key == Key.F4 && (KeyboardTools.IsCTRLDown() || KeyboardTools.IsALTDown()))
+            if ((e.Key == Key.F4 || e.Key == Key.W) && (KeyboardTools.IsCTRLDown() || KeyboardTools.IsALTDown()))
```
Comment (GerHobbelt) [inline Utilities/GUI/DualTabbedLayoutStuff/DualTabbedLayout.cs:699]: Nitpick: this selected for Ctrl+F4 and Alt+F4, now also selects for Ctrl+W and Alt+W. Be a bit stricter only allowing Ctrl+W, not Alt+W?

File: Qiqqa/Common/BackgroundWorkerDaemonStuff/BackgroundWorkerDaemon.cs [hunk 2]
```diff
@@ -58,15 +57,6 @@ private void DoMaintenance_OnceOff(Daemon daemon)
...
 
-                try
-                {
-                    AlternativeToReminderNotification.CheckIfWeWantToNotify();
-                }
-                catch (Exception ex)
-                {
-                    Logging.Error(ex, "Exception during AlternativeToReminderNotification.CheckIfWeWantToNotify");
-                }
-
```
Comment (GerHobbelt) [inline Qiqqa/Common/BackgroundWorkerDaemonStuff/BackgroundWorkerDaemon.cs:69]: I had not removed these as I was wondering how to "market" the new Qiqqa: how to enable people to find it when they don't know the name yet but are looking for something like it. 

Having a ranking at alterative.to and similar sites can help to raise awareness. I'm not a marketeer and lack better ideas than what was already implemented (yup, the twitter nags were indeed nags, but there's a balance to strike: currently Qiqqa is the most *unknown* creature out there when compared to the big boys like Zotero, Mendeley, et al -- while those serve partly different purposes, there's plenty of overlap.

For that reason, I'm inclined to keep this around in Qiqqa, despite its 'obnoxiousness'...

Better ideas welcome (might want to file them in an issue as that's a far better place for this higher-level discussion than code comments. 😄

File: Qiqqa/Common/Configuration/WebsiteAccess.cs [hunk 3]
```diff
@@ -58,8 +58,6 @@ private static bool ValidateRemoteCertificate(object sender, X509Certificate cer
         public static readonly string Url_LuceneQuerySyntax = @"http://lucene.apache.org/core/old_versioned_docs/versions/2_9_1/queryparsersyntax.html";
         public static readonly string Url_GoogleScholarBibTeXPreferences = @"http://scholar.google.com/scholar_setprefs?scis=yes&scisf=4";
 
-        public static readonly string Url_ChatAPI4Qiqqa = @"http://chatapi.qiqqa.com:8080";
```
Comment (GerHobbelt) [inline Qiqqa/Common/Configuration/WebsiteAccess.cs:61]: That one wasn't working anyway, so this is good.

While I would like to keep the number of communication channels low, there is a good reason for having a chat-like channel for *users*. While I might not be the person to "man" that kind of support desk channel, any suggestions while you're at it? 
(Gitter is an option, but there's sure to  be others. As long as it's spam-resistant -- given the !@#$%^& that's happened to the GetSatisfaction Qiqqa forum :-( )

Not to be included in the Qiqqa software, but a link to open a browser from Qiqqa to such a support/discussion channel would be good IMO.

File: Qiqqa/Documents/PDF/PDFControls/PDFReadingControl.xaml [hunk 4]
```diff
@@ -33,13 +33,6 @@
                     <local_gui:AugmentedToggleButton x:Name="ButtonInk"/>
                     <local_gui:AugmentedToggleButton x:Name="ButtonTextSentenceSelect"/>
                     <local_gui:AugmentedToggleButton x:Name="ButtonCamera"/>
-                    <local_gui:AugmentedToolBarButton x:Name="ButtonExplore"/>
-                    <local_gui:AugmentedToolBarPopup x:Name="ButtonExplorePopup">
-                        <StackPanel>
-                            <local_gui:AugmentedToolBarButton x:Name="ButtonExpedition" CaptionDock="Right" />
-                            <local_gui:AugmentedToolBarButton x:Name="ButtonExploreInBrainstorm" CaptionDock="Right" />
-                        </StackPanel>
-                    </local_gui:AugmentedToolBarPopup>
```
Comment (GerHobbelt) [inline Qiqqa/Documents/PDF/PDFControls/PDFReadingControl.xaml:42]: Just curious: why did you move it?

File: Utilities/GUI/DualTabbedLayoutStuff/DualTabbedLayout.cs (hunk 1 above)
Comment (mahfiaz) [reply Utilities/GUI/DualTabbedLayoutStuff/DualTabbedLayout.cs:699]: My bad, this was unintentional. Fixed.

File: Qiqqa/Common/BackgroundWorkerDaemonStuff/BackgroundWorkerDaemon.cs (hunk 2 above)
Comment (mahfiaz) [reply Qiqqa/Common/BackgroundWorkerDaemonStuff/BackgroundWorkerDaemon.cs:69]: I think perfect place for such information would be qiqqa.org webpage, release pages and "Tutorials & Help" panel on "Home (F1)" view.
Then again, a little nagging once in a while doesn't hurt, especially when it goes away for 12 months if you click on the link, regardless if you vote or not.
Reverted.

File: Qiqqa/Documents/PDF/PDFControls/PDFReadingControl.xaml (hunk 4 above)
Comment (mahfiaz) [reply Qiqqa/Documents/PDF/PDFControls/PDFReadingControl.xaml:42]: By its meaning citing is closer to selecting text and taking an image snapshot than to exploring with expedition. Also citing is more important than exploring (therefore closer to the beginning).

File: Qiqqa/Common/Configuration/WebsiteAccess.cs (hunk 3 above)
Comment (mahfiaz) [reply Qiqqa/Common/Configuration/WebsiteAccess.cs:61]: I am not a fan of Gitter, no login with password there. From user perspective making github.com account doesn't sound good either.
Getsatisfaction doesn't look too bad, if moderation would be possible (to delete spam etc).
IRC is fine, but is missing threads. E-mail in google groups is slightly better.
I think there is no perfect free solution.

Comment (GerHobbelt) [reply Qiqqa/Common/Configuration/WebsiteAccess.cs:61]: About Gitter: yup, feel the same. A lot of folks are using it, but it's too chaotic for my tastes TBH: no topic chains. (Same goes for IRC: I never really got into that one either.)

About GetSatisfaction: I'll nag Jimme again to give me admin access there so the link spammers can be kicked off. (Got no access rights beyond standard user there ATM)

I have set google groups several times before, and it works, still better than web forums (because it's push to single reception point which reduces the number of web spots you have to visit vs.  web forum *pull* where you have to visit the forum to see everything that's going on); filtering at high volumes is a challenge, though.

In my opinion getting a github account is about the same hurdle for a user as subscribing to a web forum (create account, join) or google group (iff you don't have a google account yet / google-recognized account, but, granted, that's a bit like not being on Facebook these days 😉 ... my kind of people 😈 

---

While many folks still believe Github is for IT/programmers, there's all this:

- https://readwrite.com/2013/11/08/seven-ways-to-use-github-that-arent-coding/
- https://medium.com/@jjmerelo/top-ten-reasons-github-is-a-great-tool-for-creative-writers-d0e8b27de71d
- https://github.com/tvanantwerp/github-for-non-programmers --> [http://github-for-non-programmers.跳.net/](http://github-for-non-programmers.xn--vt3a.net/)

but there remains some public perception work to be done before we all would agree getting a github is on the same level of effort as getting a Facebook account.

Thanks for the feedback and your work on Qiqqa.

File: Qiqqa/Documents/PDF/PDFControls/PDFReadingControl.xaml (hunk 4 above)
Comment (GerHobbelt) [reply Qiqqa/Documents/PDF/PDFControls/PDFReadingControl.xaml:42]: Wouldn't always agree with the "more important" bit there (depends on your work/task; if you're not writing papers, citing is not a much needed feature), but otherwise: yes.

Like I said: just wondering.

Picked up some of your easy changes already, will merge the rest later this week.

**Comment by GerHobbelt** [comment]
Pullreq merged with mainline.

Thanks!

File: Qiqqa/Common/Configuration/WebsiteAccess.cs (hunk 3 above)
Comment (zhangxiaoxing) [reply Qiqqa/Common/Configuration/WebsiteAccess.cs:61]: > 
> 
> I am not a fan of Gitter, no login with password there. From user perspective making github.com account doesn't sound good either.
> Getsatisfaction doesn't look too bad, if moderation would be possible (to delete spam etc).
> IRC is fine, but is missing threads. E-mail in google groups is slightly better.
> I think there is no perfect free solution.

From user perspective making github.com account doesn't sound good for _commercial packages_, I think it would be more or less OK for an academic oriented open-source software. More often than not, they will have their own repository at some point anyway.
